src/index.html

```diff
@@ -5,27 +5,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>MicroTV</title>
+  <title>MicroTV | Watch your favourite shows here</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/861ecf43a0.js" crossorigin="anonymous" defer></script>
 </head>
 
 <body>
```

Comment:
- Beautiful use of semantic tags
- Very clear and readable html structure

Reply:
Thank you very much for the review